rfcs/proposed/zip_view/README.md

```diff
@@ -0,0 +1,40 @@
+# zip_view Support for the oneDPL Range APIs with C++20
+
+## Introduction
+`std::ranges::zip_view` is powerful utility enables developers to combine two or more ranges into a single view,
+where each element is represented as a tuple containing corresponding elements from each input range.
+
+## Motivations
+`std::ranges::zip_view` is a convenient way to combine multiple ranges into a single view, where each element of
+the resulting range is a tuple containing one element from each of the input ranges. This can be particularly
+useful for iterating over multiple collections in parallel. `std::ranges::zip_view` is introduced starting with C++23,
+but there are many users who work with C++20 standard yet. So, oneDPL introduces `oneapi::dpl::ranges::zip_view`,
+which the same API and functionality as `std::ranges::zip_view`.
+
+### Key Requirements
+`oneapi::dpl::ranges::zip_view` should be:
+- compilable with C++20 version (minimum)
+- API compliant with `std::ranges::zip_view`
+- random accessible view; the "underlying" views also should be random accessible
```

Comment:
It is a stronger requirement compared to the `std::ranges::zip_view`, which "borrows" the access mode from the underlying ranges. Why is it necessary to impose it?

Reply:
Discussed it offline, and agreed that random accessibility is a mandatory requirement.

It is essential for the parallel algorithms. Moreover even algorithms with sequential execution policies will require them according to https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2024/p3179r2.html, where any algorithm accepting an execution policy is referred as a "parallel range algorithm" with further requirement of random accessibility.